src/Swashbuckle.AspNetCore.Newtonsoft/SchemaGenerator/NewtonsoftDataContractResolver.cs

```diff
@@ -201,32 +201,50 @@ private List<DataProperty> GetDataPropertiesFor(JsonObjectContract jsonObjectCon
         private static readonly Dictionary<Type, Tuple<DataType, string>> PrimitiveTypesAndFormats = new()
```

Comment:
I wonder if there's a way to refactor this here and in the other resolve to avoid the duplication.